internal/controller/datadogagent/store/store.go

```diff
@@ -225,6 +225,11 @@ func (ds *Store) Apply(ctx context.Context, k8sClient client.Client) []error {
 				objStore.SetResourceVersion(objAPIServer.GetResourceVersion())
 			}
 
+			// The CiliumNetworkPoliciesKind resource version must be set.
+			if kind == kubernetes.CiliumNetworkPoliciesKind {
```

Comment:
Can this be combined with line 224 since they both set the resource version to the same thing?